cloud-sql/sql-server/ConnectTcp.cs

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// [START cloud_sql_sqlserver_dotnet_ado_connect_tcp]
+using Microsoft.Data.SqlClient;
+using System;
+
+namespace CloudSql
+{
+    public class SqlServerTcp
+    {
+        public static SqlConnectionStringBuilder NewSqlServerTCPConnectionString()
+        {
+            // Equivalent connection string:
+            // "User Id=<DB_USER>;Password=<DB_PASS>;Server=<INSTANCE_HOST>;Database=<DB_NAME>;"
+            var connectionString = new SqlConnectionStringBuilder()
+            {
+                // Remember - storing secrets in plain text is potentially unsafe. Consider using
```

Comment:
Let's use the same wording as in the Go sample here.

Reply:
Done.